src/app/app.component.spec.ts

```diff
@@ -1,8 +1,8 @@
 import { LayoutModule } from '@angular/cdk/layout';
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-import { MatButtonModule } from '@angular/material/button';
+import { MatLegacyButtonModule as MatButtonModule } from '@angular/material/legacy-button';
```

Comment:
https://material.angular.io/guide/mdc-migration there is a migration guide.

Reply:
I think it is harder for people to migrate to MDC as it require quite a lot of CSS update.
If people were customizing your components, it would be easier for them if you don't switch for MDC yet.
Maybe you can do 2 versions, one without MDC (7.0.0) and one with MDC (7.1.0) ?

Up to you, do I migrate this PR to use MDC?

Reply:
Let's do it in next PR